Core/Libraries/Source/WWVegas/WWLib/CMakeLists.txt

```diff
@@ -169,6 +169,15 @@ target_include_directories(core_wwlib PUBLIC
     ${CMAKE_CURRENT_SOURCE_DIR}
 )
 
+target_precompile_headers(core_wwlib PRIVATE
+    always.h
+    WWCommon.h
+    STLUtils.h
+    wwstring.h
+    <windows.h>
```

Comment:
Why is this written with <> symbols? Is this correct?

Multiple times in this change.